### PR TITLE
MRU_Toggle cmd added

### PR DESCRIPTION
**Pull request by TamaMcGlinn**
Saw [this pull request](https://github.com/vim-scripts/mru.vim/pull/1) on a weird not-quite fork called [vim-scripts/mru](https://github.com/vim-scripts/mru.vim). Rebased, tested and works.

### CODE REVIEW

File: plugin/mru.vim [hunk 1]
```diff
@@ -998,6 +1012,8 @@ else
 	\ call s:MRU_Cmd(<q-args>, '', <count>)
 endif
 command! -nargs=0 MruRefresh call s:MRU_Refresh()
+command! -nargs=? -complete=customlist,s:MRU_Complete MRUToggle
+            \ call s:MRU_Toggle(<q-args>)
```
Comment (yegappan) [inline plugin/mru.vim:1017]: Thanks for the patch. Can you also update the help text with
a description of this new command?

Comment (TamaMcGlinn) [reply plugin/mru.vim:1017]: This was my first time actually documenting a vim plugin. I hope this was all that you meant.

**Comment by TamaMcGlinn** [comment]
I just read this:

```
By default, the MRU window is opened as the bottom-most window. If you are
using Vim version 8.0 and above, then you can use command modifiers like
|:topleft| or |:botright| or |:vertical| with the :MRU command to control
where the window is opened. Example: >

    :topleft MRU			" horizontally split topmost window
    :botright MRU			" horizontally split bottommost window
    :vertical topleft MRU		" vertically split far-left window
    :vertical botright MRU		" vertically split far-right window
```

and realised my mistake. I guess I need to put MRUToggle into the same if and pass the other args.

**Comment by TamaMcGlinn** [comment]
Should be good to merge now.

**Pull request merged**